proto/api/v2/compliance_results_service.proto

```
@@ -18,6 +18,7 @@ message ClusterCheckStatus {
   google.protobuf.Timestamp created_time = 3;
   string check_uid = 4;
   google.protobuf.Timestamp last_scan_time = 5;
+  ComplianceDataState data_state = 6;
```

Buf (1.72.0)

[error] cannot find `ComplianceDataState` in this scope

(COMPILE)